contracts/child/bor/BorValidatorSet.sol

```diff
@@ -0,0 +1,383 @@
+pragma solidity ^0.5.2;
```

Comment:
Do we need to move bor contracts (other than `StateSyncerVerifier`) in contracts repo as well? @jdkanani

Reply:
either we keep'em in a separate repo and publish npm package and write tests in different repo or keep in a single repo with monolithic code for smart contracts. or even move back to genesis contracts repo. I feel no harm in keeping here. we have plasma, staking, and add genesis contracts too here.

Reply:
where did this move to?